smart-cart/app/controllers/lists_controller.rb

```diff
@@ -5,6 +5,8 @@ class ListsController < ApplicationController
   # GET /lists or /lists.json
   def index
     # @lists = List.all
+    @list = []
+    @current_list = $list
```

Comment:
now that the app is sitting on a database, we should remove these and use the actual List resource to build out a list instead the arrays

Reply:
yeah i like that idea, i can help out with that second task

Reply:
okay nice, @jdexcond i can pair with you on it if you get stuck or anything!!
@alexrpark if thats cool w you, i'll just approve this PR and make the second task

Reply:
@e33lin sounds good, I'll also help w the second part!